website/src/types/config.ts

```diff
@@ -27,6 +27,19 @@ export const customDisplay = z.object({
     displayGroup: z.string().optional(),
 });
 
+/**
+ * RangeOverlapSearch to configure on two fields that together allow to query
+ * For an overlap of a search range and a targer range
+ */
+export const rangeOverlapSearch = z.object({
+    /**
+     * specify rangeName in both (upper and lower) fields to link them later.
```

Comment:
Still not a fan of using 3 lines for comments when 1 would do

Reply:
I can try and remember, would be nice to update the linter/formatter if possible with this.